src/SFA.DAS.Roatp.CourseManagement.Application/ProviderStandards/Commands/DeleteCourseLocations/DeleteCourseLocationsCommandHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MediatR;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Roatp.CourseManagement.Domain.Interfaces;
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Roatp.CourseManagement.Application.Standards.Commands.DeleteCourseLocations
+{
+    public class DeleteCourseLocationsCommandHandler : IRequestHandler<DeleteCourseLocationsCommand, Unit>
+    {
+        private readonly ILogger<DeleteCourseLocationsCommandHandler> _logger;
+        private readonly IApiClient _apiClient;
+
+        public DeleteCourseLocationsCommandHandler(ILogger<DeleteCourseLocationsCommandHandler> logger, IApiClient apiClient)
+        {
+            _logger = logger;
+            _apiClient = apiClient;
+        }
+
+        public async Task<Unit> Handle(DeleteCourseLocationsCommand request, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation("Command triggered to delete regional locations for ukprn:{ukprn} LarsCode:{larscode} from user:{userid}", request.Ukprn, request.LarsCode, request.UserId);
```

Comment:
the comments in this state it's about regional locations being deleted, but it does read from the command name etc to be all courselocations (provider, regional, national).   If it's all please amend comments, if it's just regions, leave as is.

Reply:
Updated log